main.go

```diff
@@ -383,8 +383,8 @@ func getViolations(test windup.Ruletest) ([]hubapi.RuleSet, error) {
 	return violations, nil
 }
 
-func runTestRule(rule windup.When, violations []hubapi.RuleSet) bool {
```

Comment:
I named it violations because it's the output of running a ruleset, since we use ruleset throughout the code to mean a set of rules to run I think the distinction is useful